var/spack/repos/builtin/packages/gsl/package.py

```diff
@@ -55,3 +55,8 @@ def configure_args(self):
             configure_args.append("CBLAS_LIBS=%s" % self.spec["blas"].libs.ld_flags)
 
         return configure_args
+
+    # cmake looks for GSL_ROOT_DIR to find GSL so this helps pick the spack one
+    # when there are multiple installations (e.g. a system one and a spack one)
+    def setup_run_environment(self, env):
```

Comment:
Nitpick, but the comment should refer to env.set, not the `setup_run_environment` method (that might be extended later to have other purposes too):
```suggestion
    def setup_run_environment(self, env):
        # cmake looks for GSL_ROOT_DIR to find GSL so this helps pick the spack one
        # when there are multiple installations (e.g. a system one and a spack one)
```